### ISSUE TITLE
Dependency error

### ISSUE THREAD

**Issue by absolutejam** [sensu-plugins/sensu-plugins-google-spreadsheet#3]
I was getting error `cannot load such file -- google/api_client (LoadError)`, and had to manually install `google-api-client -v 0.8.1` (has to be `< 0.9`, unless the `require` statement is altered as per https://github.com/google/google-api-ruby-client/blob/master/MIGRATING.md).

**Comment by absolutejam** [comment]
Looking at this some more, I think this handler won't actually work as it seems to use a lot of out-dated gems and one of them in particular (https://github.com/gimite/google-drive-ruby) needs to be updated to use the newer Google API.

**Comment by mbbroberg** [comment]
Yeah good point. On the bright side there's a migration path for that gem in particular: https://github.com/gimite/google-drive-ruby/blob/master/MIGRATING.md
